### PR TITLE
Add Citizen profile page

### PR DESCRIPTION
**Pull request by aahna-ashina**
## Dework Task

https://app.dework.xyz/nation3/nationcred-p?taskId=246b633c-5075-4ba4-a6d1-e8e62bd96ffc

## Screenshots

> Screenshot 2022-11-17 at 3 48 12 PM

> Screenshot 2022-11-18 at 9 57 55 AM

> Screenshot 2022-11-18 at 5 37 32 PM

### CODE REVIEW

File: pages/profile/[ethAddress].tsx [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    if (!chart) {
+      return
+    }
+
+    let colorGradient = chart.ctx.createLinearGradient(0, 0, 0, 400)
+    colorGradient.addColorStop(0, 'rgba(213, 163, 152, 0.2)')
+    colorGradient.addColorStop(1, 'rgba(213, 163, 152, 0.8)')
+
+    // Fetch data from datasets repo
+    const sourceCredFileUrl: string = `https://raw.githubusercontent.com/nation3/nationcred-datasets/f9e5b8d30b969815a918de619fa4e89a1c800ce3/data-sources/sourcecred/output/sourcecred-${profile.ethAddress}.csv`
```
Comment (aahna-ashina) [inline pages/profile/[ethAddress].tsx:56]: @johnmark13 Using the data from your PR here.  Once merged, we can update this URL.

**Review by johnmark13 — APPROVED** [review]
I think this is super cool. The chart colors at the moment slightly limit legibility/determining different streams, but that seems a small thing for now.

**Comment by aahna-ashina** [comment]
> The chart colors at the moment slightly limit legibility/determining different streams, but that seems a small thing for now.

@johnmark13 Yes, we can easily adjust the colors.  Also note that you can hover points in the chart to see details:

> Screenshot 2022-11-18 at 9 11 04 AM

**Comment by aahna-ashina** [comment]
@johnmark13 Added Dework chart:

> Screenshot 2022-11-18 at 9 57 55 AM

**Review by okhaimie-dev — APPROVED** [review]
Nice! 

Maybe we can also add pagination or a custom search bar on the directory / default page for better UX in the future.

**Comment by aahna-ashina** [comment]
> Maybe we can also add pagination or a custom search bar on the directory / default page for better UX in the future.

@okhaimie-dev Great ideas!  Added those here:

- https://github.com/nation3/citizen-directory/issues/12
- https://github.com/nation3/citizen-directory/issues/13

**Comment by xPi2** [comment]
Why the "Loading profile..." placeholder?

**Comment by aahna-ashina** [comment]
@xPi2 Yes.  Work in progress 😄

**Review by xPi2 — DISMISSED** [review]

**Review by luisivan — APPROVED** [review]

**Pull request merged**